[addons] add for test purpose only a script for debian package generation

AlwinEsch opened the pull request:
This script is meant to test at home for the fast debian added code.

## Description
He is deposited in "./xbmc/addons/kodi-addon-dev-kit/tools", as this is
supposed to be purely for addons. Furthermore, "kodi-addon-dev-kit" folder
should be as independent as possible from Kodi.

<!--- Provide a general summary of your change in the Pull Request title above -->
<!--- Describe your change in detail here -->

## Motivation and Context
To be able to test quickly at home, which is approximately necessary for Jenkins.

## How Has This Been Tested?
```
./debian-addon-package-test.sh $HOME/Development/Kodi/addons/screensaver/screensaver.stars
./debian-addon-package-test.sh $HOME/Development/Kodi/addons/screensaver/screensaver.stars --clean
```
Log: https://pastebin.com/ZxSnbe4u

<!--- Please describe in detail how you tested your change -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc -->

## Screenshots (if appropriate):

## Types of change
<!--- What type of change does your code introduce? Put an `x` in all the boxes that apply like this: [X] -->
- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [X] **None of the above** (please explain below)
  Independent script

## Checklist:
<!--- Go over all the following points, and put an `X` in all the boxes that apply like this: [X] -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

wsnipex commented on xbmc/addons/kodi-addon-dev-kit/tools/debian-addon-package-test.sh:
```diff
@@ -0,0 +1,141 @@
+  rm -f ${BASE_DIR}/debian/debhelper-build-stamp
+  rm -f ${BASE_DIR}/debian/files
+  rm -f ${BASE_DIR}/debian/*.log
+  rm -f ${BASE_DIR}/debian/*.substvars
```
better cd to base_dir then rm relative paths

wsnipex commented on xbmc/addons/kodi-addon-dev-kit/tools/debian-addon-package-test.sh:
```diff
@@ -0,0 +1,141 @@
+echo "Create needed compressed source code file: ${BASE_DIR}/../${ORIGTARBALL}.tar.gz"
+
+# git archive --format=tar.gz -o ${BASE_DIR}/../${ORIGTARBALL} HEAD # Unused git, leaved as optional way
+tar -zcvf ${BASE_DIR}/../${ORIGTARBALL} * # Used to prevent on code changed for test a git commit
```
--exclude=.git/*

AlwinEsch replied:
Thanks! Have extended it a bit and .gitignore becomes also now scanned to prevent not wanted parts.

AlwinEsch committed:
[addons] add for test purpose only script for debian package generation

This script is meant to test at home for the fast debian added code.

He is deposited in "./xbmc/addons/kodi-addon-dev-kit/tools", as this is
supposed to be purely for addons. Furthermore, "kodi-addon-dev-kit" folder
should be as independent as possible from Kodi.
